### ISSUE TITLE
REMOVE all environemnt setup tools.

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs.github.io#597]
Currently you can't use the `Makefile` unless you are running linux (one that has aptitude installed).

**We are at a serious issue where there are plenty of ways to setup this environment and they are all wrong in some way.**

I propose we remove all of them and opt for a simpler approach which is to provide good documentation on what is needed.

I'm okay with the devcontainer setup being kept as it works in a codespace environment. That said it's incredibly slow and needs to be simplified.

Tasks in this Epic:

- [x] #604
- [x] #605
- [x] #606

**Comment by kjaymiller** [comment]
@dragid10, @TimOsahenru I know y'all put in work trying to get things figured out. 

I won't create the other issues until I get a reaction from at least one of y'all. This portion of things is untenable.

**Comment by dragid10** [comment]
Yeah I think having the good documentation is the most accessible solution to this

**Comment by TimOsahenru** [comment]
> Currently you can't use the `Makefile` unless you are running linux (one that has aptitude installed).
> 
> **We are at a serious issue where there are plenty of ways to setup this environment and they are all wrong in some way.**
> 
> I propose we remove all of them and opt for a simpler approach which is to provide good documentation on what is needed.
> 
> I'm okay with the devcontainer setup being kept as it works in a codespace environment. That said it's incredibly slow and needs to be simplified.
> 
> Tasks in this Epic:
> 
> * [ ]  Ensure Make is not being used at all
> * [ ]  Remove makefile and gitpod
> * [ ]  Rebuild DevContainer to be a base installation of python and ruby and then run pip install and ruby bundle and playwright install with deps

> * [ ]  Simplify/update contributing guide

**Comment by TimOsahenru** [comment]
> Yeah I think having the good documentation is the most accessible solution to this

Please create an issue on this. Thanks

**Comment by TimOsahenru** [comment]
I think this issue has bee resolved. Closing it
